### DISCUSSION TITLE
Using Cake to build and possible package Textrude

### DISCUSSION THREAD

**Discussion by highstreeto** [NeilMacMullen/Textrude#48] [Ideas]
Short summary, what *Cake* is: [Cake](https://cakebuild.net/) is a build tool based on .NET, where the build scripts are written in an C# DSL. With this it should be easier to build, run tests, bundle / package, etc. in one script.

In my opinion, it is quite a good tool for build automation and works especially well in .NET projects, as the same language is used
But the integration with Visual Studio is very light, as only a single task can be run from the script, but not the build action or unit test running is changed.

Thoughts? (if they are no objections, I would open a PR)

**Comment by NeilMacMullen** [comment]
Yes I have a passing familiarity with Cake (looked at a while back on another project).  Sounds like an excellent idea - go for it ! :-)

**Comment by NeilMacMullen** [comment]
Thanks again - this really is very cool! :-)
